vonk/releasenotes/roadmap.rst

```diff
@@ -0,0 +1,57 @@
+.. _vonk_roadmap:
+
+Vonk Roadmap
+============
+
+This page lists the features and changes we have planned for the foreseeable future. This planning is volatile, so changes will happen. We will update this page accordingly. You are also very welcome to provide input to us on features or fixes that you would like to see prioritized. 
+
+Disclaimer: No rights can be derived from this roadmap.
+
+2020
+----
+
+Q2
+^^
+
+* FHIR Mapping for CCDA to FHIR
```

Comment:
Will the maps be published, or just the fact that we'll have support for running your maps?

Reply:
Only support for your own maps, I don't think that HeC will allow a publication.

Reply:
We need to talk to them about that, because they got the maps for free from the community and I don't think they realise that.

Also this statement needs to be reworded better, it implies you get the fhir mapper and you get ccda support

Reply:
```suggestion
* FHIR Mapping improvements to support odd features in logical models like those for CCDA
```

Reply:
@vadi2 : how about the suggestion above?

Reply:
That looks good @cknaap.